op-supervisor/supervisor/backend/db/db.go

```diff
@@ -86,6 +87,10 @@ type ChainsDB struct {
 	// cross-safe: index of L2 blocks we know to only have cross-L2 valid dependencies
 	crossDBs locks.RWMap[types.ChainID, CrossDerivedFromStorage]
 
+	// gethevent Feeds allow for subscription to events in the DB
+	l2FinalitySubscription locks.RWMap[types.ChainID, gethevent.FeedOf[eth.BlockID]]
+	crossSafeSubscription  locks.RWMap[types.ChainID, gethevent.FeedOf[types.DerivedPair]]
```

Comment:
map values should be pointers, or it shallow-copies the event-feed struct during reorganization of the map.